src/Application/Authorisation/Suggest/Resource.elm

```diff
@@ -170,3 +192,44 @@ resourceItem additionalClasses nodes =
             ]
             nodes
         ]
+
+
+rawLabel nodes =
+    Html.div
+        [ T.flex
+        , T.items_center
+        ]
+        nodes
+
+
+rawResource nodes =
```

Comment:
This repeats many of the styles we already have, but I wasn't quite sure how to refactor it. Open to suggestions and happy to rework this!

Reply:
Yep! It looks like:

![permissions](https://user-images.githubusercontent.com/7957636/118878584-92aeab00-b8a4-11eb-8402-dc471a0f6d85.png)